pkg/handler/payload.go

```diff
@@ -267,3 +282,48 @@ func parseImageFormDataTextToImageInputs(req *http.Request) (textToImageInput []
 		Samples:  1,
 	}}, nil
 }
+
+func parseTextFormDataTextGenerationInputs(req *http.Request) (textGeneration []triton.TextGenerationInput, err error) {
+	prompts := req.MultipartForm.Value["prompt"]
+	if len(prompts) != 1 {
+		return nil, fmt.Errorf("invalid input")
```

Comment:
nit: maybe a more specific message such as "the model do not support batch, only single prompt is allowed"

Reply:
nice catch, revised